CHANGES.md

```diff
@@ -1,6 +1,422 @@
 Release Changes List
 ====================
 
+version 2.0
+===========
+
+released March 21, 2024
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* Chapel's core language and library features are now considered stable!
+* added `chpl-language-server`, a new LSP-based tool for VSCode/vim/... users
+* added new 'ParallelIO', 'DistributedBag' and 'Zarr' package modules
```

Comment:
I view DistributedBag as significantly reworked rather than new; but you could say it is a new implementation